### PR TITLE
Test custom payment source

### PR DESCRIPTION
**Pull request by alepore**
This PR adds some payment sources tests using a custom payment source instead of the common `Spree::CreditCard` source.
A generator is included to setup the dummy app with the custom resources.

### CODE REVIEW

**Comment by alepore** [comment]
@tvdeyen 👍 just moving it to `lib/_generators` does the trick

File: spec/features/checkout_spec.rb [hunk 1]
```diff
@@ -58,4 +57,57 @@
       expect(page).to have_current_path(spree.order_path(Spree::Order.last))
     end
   end
+
+  context "with custom payment sources" do
+    let(:roman) { create(:roman_payment_method) }
+
+    let!(:roman_wallet) do
+      create(:roman_wallet, name: "Naughtius Maximus", payment_method: roman)
```
Comment (mamhoff) [inline spec/features/checkout_spec.rb]: https://www.youtube.com/watch?v=OuALwLhUmKY

File: Rakefile [hunk 2]
```diff
@@ -32,6 +32,6 @@ end
```
Comment (mamhoff) [inline Rakefile]: Typo in the commit message `s/don0t/don't`

File: spec/features/checkout_spec.rb (hunk 1 above)
Comment (alepore) [inline spec/features/checkout_spec.rb]: 💯

**Pull request merged**